client/my-sites/hosting/sftp-card/index.js

```diff
@@ -24,64 +26,104 @@ import {
 } from 'state/data-getters';
 
 const SFTPCard = ( { translate, username, password, errorCode, siteId, loading } ) => {
+	const [ isCopied, setIsCopied ] = useState( false );
+	const usernameIsCopied = isCopied === 'username';
+	const passwordIsCopied = isCopied === 'password';
+
 	const sftpData = {
 		[ translate( 'URL' ) ]: 'sftp1.wordpress.com',
 		[ translate( 'Port' ) ]: 22,
-		[ translate( 'Username' ) ]: username,
 	};
 
 	return (
-		<Card>
-			<div className="sftp-card__icon-col">
+		<Card className="sftp-card">
+			<div className="sftp-card__icon">
 				<MaterialIcon icon="cloud" size={ 32 } />
 			</div>
-			<div>
+			<div className="sftp-card__body">
 				<CardHeading>{ translate( 'SFTP Information' ) }</CardHeading>
-				<p>{ translate( "Access and edit your website's files directly using an FTP client." ) }</p>
-				{ password && (
-					<div className="sftp-card__callout-box">
-						<p>{ translate( 'Your new password for your sftp user is:' ) }</p>
+				{ errorCode === 404 ? (
```

Comment:
For folks 👀 We can further tidy the data pattern in a different PR. Basically we have a default retry policy that will, retry 3 times on any GET requests for a non 2xx status code. In this case the 404 is a clear answer, so we either have a choice of updating the API response or modifying client behavior.

Note that we need to also still need to handle non 404 cases (5xx)